compiler/rustc_middle/src/ty/consts/int.rs

```diff
@@ -109,3 +112,258 @@ impl std::fmt::Debug for ConstInt {
         }
     }
 }
+
+// FIXME: reuse in `super::int::ConstInt` and `Scalar::Bits`
```

Comment:
This is used in `ConstInt` and `Scalar`, so I am confused by this FIXME.

Reply:
that is an outdates FIXME. The early version of this PR did not try to do all of this at the same time.